content/tutorial/tutorial.md

```diff
@@ -260,13 +265,13 @@ If you click on a Square now, you should see an alert in your browser.
 >}
 >```
 >
->Notice how with `onClick={() => alert('click')}`, we're passing *a function* as the `onClick` prop. React will only call this function after a click. Forgetting `() =>` and writing `onClick={alert('click')}` is a common mistake, and would fire the alert every time the component re-renders.
+>ध्यान दें कि `onClick = {() => alert('click')}` के साथ, हम `onClick` props के रूप में *एक फ़ंक्शन* पास कर रहे हैं। यह केवल एक क्लिक के बाद फायर करता है। भूल जाना `() =>` और ऐसे `onClick = {alert('click')}` लिखना एक सामान्य गलती है, और हर बार कौम्पोनॅन्ट फिर से रेंडर करने पर अलर्ट दिखायेगा।
 
-As a next step, we want the Square component to "remember" that it got clicked, and fill it with an "X" mark. To "remember" things, components use **state**.
+अगले चरण के रूप में, हम Square कौम्पोनॅन्ट को "याद रखना" चाहते हैं कि यह क्लिक हो गया, और इसे "X" चिह्न के साथ भरें। चीजों को "याद रखने" के लिए, कौम्पोनॅन्ट **state** का उपयोग करते हैं।
```

Comment:
"याद रखना" चाहते हैं कि यह क्लिक हो गया does not seem grammatically correct

Reply:
का उपयोग करते हैं। -> ka upyog kiya jata hai

Reply:
> "याद रखना" चाहते हैं कि यह क्लिक हो गया does not seem grammatically correct

`हम चाहते हैं की स्क्वैर कॉम्पोनेन्ट याद रखे की वह क्लिक हुआ था, और खुद को "X" मार्क से भर ले` ?

Reply:
cool